api/services/EmailService/email.service.js

```diff
@@ -1,8 +1,8 @@
+const EmailTemplates = require('./email.templates');
 
 const mailgun = require('mailgun-js')({
   apiKey: process.env.MAILGUN_API_KEY,
   domain: process.env.MAILGUN_DOMAIN,
-  host: 'api.eu.mailgun.net',
```

Comment:
Hopefully that doesn't kill production @gustavlrsn

Reply:
I think it might kill production, maybe we can add a env variable `MAILGUN_HOST`?